server/app/views/admin/migration/AdminExportView.java

```diff
@@ -84,16 +89,21 @@ private DomContent createProgramSelectionForm(
     return div()
         .with(
             form()
+                .withId(PROGRAM_EXPORT_FORM_ID)
                 .attr("hx-encoding", "multipart/form-data")
                 .attr("hx-post", routes.AdminExportController.hxExportProgram().url())
                 .attr("hx-target", "#" + AdminExportViewPartial.PROGRAM_JSON_ID)
                 .attr("hx-swap", "outerHTML")
                 .with(makeCsrfTokenInputTag(request))
                 .with(fields)
-                .with(submitButton("Generate Json").withClasses(ButtonStyles.SOLID_BLUE, "mb-10")));
+                .with(
+                    submitButton("Generate JSON")
+                        .withId(GENERATE_JSON_BUTTON_ID)
+                        .withCondDisabled(true)
```

Comment:
suggestion (non-blocking):  I think it would be clearer to use `.isDisabled()` rather than having a condition that is just set to `true`.

Reply:
good call! done.